tests/bootstrap_strings.py

```diff
@@ -1,7 +1,4 @@
-from pathlib import Path
-
 TEST_MANIFEST = '# The manifest for the "dev" environment.\n# Read the full specification for the "Environment" type at:\n#  https://aws.github.io/copilot-cli/docs/manifest/environment/\n\n# Your environment name will be used in naming your resources like VPC, cluster, etc.\nname: test\ntype: Environment\n\n# Import your own VPC and subnets or configure how they should be created.\n# network:\n#   vpc:\n#     id:\n\n# Configure the load balancers in your environment, once created.\n\nhttp:\n  public:\n    certificates:\n      - ACM-ARN-FOR-test.landan.cloudapps.digital\n#   private:\n\n\n# Configure observability for your environment resources.\nobservability:\n  container_insights: true\n'
```

Comment:
Just pointing out, you raised an issue with the env names in these test files in case you did want to remediate here.

Reply:
🤣 Brilliant. Touché. I walked right into that one. Let me sort it out so that we never have to speak of this again!

Reply:
I kind of found a bug, but it's only the environment hardcoded in a comment in the template, so wouldn't break anything, just confuse anyone who actually read it.

Reply:
Sorted I think.